server/execution/web/notification.py

```diff
@@ -0,0 +1,27 @@
+from flask import Blueprint, request
+
+from execution import run
+from web.api.register import admin_only
+
+web_api = Blueprint("web_api-notification", __name__)
+
+
+@admin_only
+@web_api.post("/notifications/post")
+def add_notification_to_execution():
+    """
+    Endpoint for adding a notification to an execution via the execution id.
+    Each parameter is set as form parameter.
+    Returns OK for successful requests or BAD_REQUEST in case of invalid parameters
+    """
+    form = request.form
+
+    try:
```

Comment:
After #125 is merged you can use the `@required` decorator which handles form and query parameter validation.